tests/crypto/enclave/host/host.c

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+// Suppress warnings against using _open() instead of _sopen_s().
+#pragma warning(disable : 4996)
```

Comment:
May need win32 ifdef

Reply:
It looks like this wasn't fixed yet, reopening

Reply:
Oops, just fixed it